### PR TITLE
Advanced Telemetry Parsing 1: Maybe we should do more than just check bans

### PR DESCRIPTION
**Pull request by francinum**
## About The Pull Request

Utilizes the data we have available to us via TGUI Telemetry far more effectively.

## Why It's Good For The Game

More datapoints.
Privacy is a joke.
Parse everything.

## Changelog
:cl:
admin: Additional data about players can be found in the Player Panel.
/:cl:

### CODE REVIEW

**Comment by DatBoiTim** [comment]
Beestation Panopticon when 😳 ?

**Comment by utilbird** [comment]
1984

**Comment by mat-lap** [comment]
literally 1984

**Comment by BrumalLake** [comment]
big brother comes to beestation

**Comment by ZombiesVsPlants** [comment]
image

**Comment by OceanFish1** [comment]
literally 1984

**Comment by ghost** [comment]
Literally 1984 surveillance state 😔

**Comment by PowerfulBacon** [comment]
2584

File: code/modules/tgui_panel/telemetry.dm [hunk 1]
```diff
@@ -7,20 +7,32 @@
...
  * Maximum time allocated for sending a telemetry packet.
  */
 #define TGUI_TELEMETRY_RESPONSE_WINDOW 30 SECONDS
 
+/// Telemetry statuses
+#define TGUI_TELEMETRY_STAT_NOT_REQUESTED 0 //Not Yet Requested
+#define TGUI_TELEMETRY_STAT_AWAITING      1 //Awaiting request response
+#define TGUI_TELEMETRY_STAT_ANALYZED      2 //Retrieved and validated
+#define TGUI_TELEMETRY_STAT_MISSING       3 //Telemetry response window miss without valid telemetry
+#define TGUI_TELEMETRY_STAT_OVERSEND      4 //Telemetry was already processed but was repeated
```
Comment (Archanial) [inline code/modules/tgui_panel/telemetry.dm]: those are missing undef

**Comment by ike709** [comment]
ITT: Everyone shows that they have no idea what this actually does.

**Comment by Misha600** [comment]
PR: Adds CIA Backdoor

**Comment by ghost** [comment]
> ITT: Everyone shows that they have no idea what this actually does.

I can't ban evade more easily now :(

**Comment by MCterra10** [comment]
This is audacity all over again except you can't opt out

**Comment by AffectedArc07** [comment]
Oh no players are whining about fingerprinting that helps identify ban evaders

Oh nooooooooooooooooooooooooooooo


For context, all of this shit was already logged on you. This just makes it easier for admins to find it.

**Comment by utilbird** [comment]
1984

**Comment by MCterra10** [comment]
1984

**Pull request merged**